docs/index.html

```diff
@@ -15,23 +15,45 @@
     <style type="text/css">
 	    .fixed {
   		    position: fixed;
-            top: 0;
-            width: 100%;
+  		    top: 0;
   		    padding: 5px;
   		    border: 2px solid;
 	    }
 	    
 	    body, h2 {
```

Comment:
Don't use html tag selectors in your custom css.  Create a class that describes your change and apply that class to the element. 
Even better, use bootstrap classes to the HTML elements to apply the desired spacing.  https://getbootstrap.com/docs/4.0/utilities/spacing/